castle/cms/browser/controlpanel/templates/status.pt

```diff
@@ -25,15 +25,35 @@
     </div>
 
     <div id="content-core">
-        <div tal:content="structure python:view.docsplit()"/>
+        <div tal:condition="python:view.docsplit()">
+            <span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: OK
+        </div>
     </div>
     <div id="content-core">
-    <div tal:content= "structure python: view.elasticsearch()" />
+        <div tal:condition="python:view.docsplit() == False">
+            <span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Docsplit: ERROR <span tal:content = "structure python: view.docsplitError()"/>
+        </div>
     </div>
     <div id="content-core">
-        <div tal:content= "structure python: view.celery()" />
+        <div tal:condition="python:view.elasticsearch()">
```

Comment:
same comment here about using tal:define to store view.elasticsearch()'s return value, and similarly below for view.celery()